src/containers/StructurePage/resourceComponents/StructureResources.jsx

```diff
@@ -236,22 +241,36 @@ export class StructureResources extends React.PureComponent {
           currentTopic={currentTopic}
           status={topicStatus}
         />
-        {resourceTypes.map(resourceType => {
-          const topicResource =
-            topicResources.find(resource => resource.id === resourceType.id) || {};
-          return (
-            <ResourceGroup
-              key={resourceType.id}
-              resource={resourceType}
-              topicResource={topicResource}
-              params={this.props.params}
-              refreshResources={this.getTopicResources}
-              locale={locale}
-              currentSubject={currentSubject}
-              disable={resourceType.disabled}
-            />
-          );
-        })}
+        {topicResources.length > 0 && grouped === 'ungrouped' && (
```

Comment:
Ser ut som om dette er tilstrekkelig til å alltid vise linje for å legge til ressurs til emne.
```suggestion
        {grouped === 'ungrouped' && (
```